pommerman/network/ionc/constants.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+"""IonClient Constants"""
+
+# Imports
```

Comment:
Remove

Reply:
I can remove the `#Imports` but not the docstring or else pylint will error out.

Reply:
I'll make the docstring better in describing the file.

Reply:
^^